tests/configs/configs_env_validate_test.py

```diff
@@ -0,0 +1,323 @@
+import os
+from typing import Optional
+import pytest
+import requests
+
+from node_cli.configs.env import (
+    absent_params,
+    load_env_file,
+    build_params,
+    populate_params,
+    get_env_config,
+    validate_params,
+    validate_env_type,
+    validate_env_alias_or_address,
+    validate_contract_address,
+    validate_contract_alias,
+    get_chain_id,
+    get_network_metadata,
+    ContractType,
+    ALLOWED_ENV_TYPES,
+)
+from node_cli.utils.exit_codes import CLIExitCodes
+
+
+# =============================================================================
+# Helper fake response for patching requests.get in network helpers
+# =============================================================================
+class FakeResponse:
+    def __init__(self, status_code: int, json_data: Optional[dict] = None):
+        self.status_code = status_code
+        self._json_data = json_data or {}
+
+    def json(self):
+        return self._json_data
+
+
+# =============================================================================
+# Tests for absent_params
+# =============================================================================
+class TestAbsentParams:
```

Comment:
all tests in skale.py, node-cli, skale-admin, and other repos use a function-based approach for pytest, not classes.
also, the pytest was designed with function-based tests as the primary pattern.

update new tests to use isolated functions instead of classes
if functions in a class do not use any shared resources and are just used to separate logical pieces, it make sense to move them to the separate files instead